### PR TITLE
Rankout toggle UI [CU-3r8be0j]

### PR DESCRIPTION
**Pull request by CodeRedJack**
Toggle added to Player Table in Admin, pops confirmation window before toggling.

This was FE only, not currently connected to any mutation

### CODE REVIEW

File: clients/web/src/lib/components/molecules/scrims/AdminPlayerTable.svelte [hunk 1]
```diff
@@ -7,19 +7,50 @@
...
-            <td/>
+            <th>RankOuts: <button
+                    class="btn btn-outline btn-accent btn-sm "
+                    on:click= {async () => {
+                        await toggleRankOuts(!rankOutsEnabled);
+                    }}
+            >
+                <span class="justify-center">
+                {#if rankOutsEnabled}
+                    <label class="flex"> On </label>
```
Comment (bwpikaard) [inline clients/web/src/lib/components/molecules/scrims/AdminPlayerTable.svelte:47]: Remove space before and after `On`

File: clients/web/src/lib/components/molecules/scrims/AdminPlayerTable.svelte [hunk 2]
```diff
@@ -7,19 +7,50 @@
...
+                    {:else}
+                    <label class="flex"> Off </label>
```
Comment (bwpikaard) [inline clients/web/src/lib/components/molecules/scrims/AdminPlayerTable.svelte:49]: See previous

**Review by bwpikaard — CHANGES_REQUESTED** [review]

File: clients/web/src/lib/components/molecules/scrims/AdminPlayerTable.svelte (hunk 1 above)
Comment (CodeRedJack) [reply clients/web/src/lib/components/molecules/scrims/AdminPlayerTable.svelte]: Changed

**Comment by 0c370t** [comment]
Task linked: [CU-3r8be0j Toggle Salaries/Rankouts (FE)](https://app.clickup.com/t/3r8be0j)

**Review by zachstence — APPROVED** [review]

**Review by bwpikaard — CHANGES_REQUESTED** [review]
Connect to backend.

**Comment by bwpikaard** [comment]
#225

**Pull request closed without merging**